src/de/grobox/blitzmail/NoteActivity.java

```diff
@@ -50,10 +51,7 @@ protected void onCreate(Bundle savedInstanceState) {
 		.setTitle(R.string.note_name)
 		.setNegativeButton(android.R.string.cancel, new DialogInterface.OnClickListener() {
 			public void onClick(DialogInterface dialog, int id) {
-				textView.setText(null);
-				saveText(null);
-
-				finish();
+				dialog.cancel();
```

Comment:
Will the activity stay and overlay the screen when the dialog is only cancelled?

Reply:
Ah I guess the CancelListener then takes care of finishing it, right?

Reply:
> Ah I guess the CancelListener then takes care of finishing it, right?

Yes. Since this is the cancel button I figured it would be fine to delegate to .cancel() but I can certainly put back `textView.setText("");finish();` if you feel like it's more explicit?